src/templates/config/.terrahub.json

```
@@ -2,5 +2,6 @@
   "format": "yml",
   "token": false,
   "retryCount": 2,
-  "usePhysicalCpu": false
+  "usePhysicalCpu": true,
```

revert this change, please
```suggestion
  "usePhysicalCpu": false,
```